docs/content/en/docs/user-guide/configuration-reference.md

```diff
@@ -93,13 +93,15 @@ spec:
 ``` yaml
 apiVersion: pipecd.dev/v1beta1
 kind: ECSApp
+input:
```

Comment:
This field should be in `spec`.

Reply:
Nice catch, thx 🙏

Reply:
addressed on https://github.com/pipe-cd/pipe/pull/2096/commits/5631c195023f31cb333990d1c99bc09c21129ac6 🙏